polar2grid/compare.py

```diff
@@ -512,7 +528,8 @@ def main(argv=sys.argv[1:]):
 
     levels = [logging.ERROR, logging.WARN, logging.INFO, logging.DEBUG]
     logging.basicConfig(level=levels[min(3, args.verbosity)])
-    compare_kwargs = {"shape": tuple(args.shape), "dtype": args.dtype, "variables": args.variables}
+    compare_kwargs = {"shape": tuple(args.shape), "dtype": args.dtype, "variables": args.variables,
+                      "file_type": args.file_type}
```

Comment:
Good catch. Could you verify that if this is not provided that `args.file_type` is `None` (versus `[]`). If it is `[]` then it won't get detected later on when we do `file_type is None`.